docs/tutorials/advanced-techniques-for-qaoa.ipynb

```diff
@@ -1186,7 +1186,7 @@
    "id": "99e70f6e-f794-4dcf-b991-5da71496ba53",
    "metadata": {},
    "source": [
-    "The following graph is the results of the standard expectation value."
+    "The following graph is the result of the standard expectation value."
```

Comment:
I don't see a graph immediately following this sentence. The code that follows seems to output a list?

Reply:
I would rephrase it as "The following cost is the result of the standard expectation value."